frontend/src/scenes/cohorts/CohortEdit.tsx

```diff
@@ -147,6 +159,32 @@ export function CohortEdit({ id }: CohortLogicProps): JSX.Element {
                                 )}
                             </LemonField>
                         </div>
+                        {!isNewCohort && !cohort?.is_static && (
+                            <div className="max-w-70 w-fit">
+                                <div className="flex gap-1 flex-col">
+                                    <LemonLabel>Last calculated</LemonLabel>
+                                    {cohort.last_calculation ? (
+                                        <div className="flex flex-1 flex-row gap-1">
+                                            <TZLabel time={cohort.last_calculation} />
+                                            {cohort.errors_calculating ? (
+                                                <Tooltip
+                                                    title={
+                                                        "The last attempted calculation failed. This means your current cohort data can be stale. This doesn't affect feature flag evaluation."
+                                                    }
+                                                >
+                                                    <div className="text-danger">
+                                                        <IconErrorOutline className="text-danger text-xl shrink-0" />
+                                                    </div>
+                                                </Tooltip>
+                                            ) : null}
+                                        </div>
+                                    ) : (
+                                        <div className="text-s">N/A</div>
```

Comment:
N/A looks really raw – in this case I propose we show "In progress…" if calculation is ongoing, and "Yet to calculate" if for some reason there's no last calculation AND calculation isn't happening right now

Reply:
reasonable, thanks, will update!